index.js

```diff
@@ -84,17 +86,23 @@ function start(clusterProcessId = 0) { // eslint-disable-line max-statements
 }
 
 function startMaster() {
-  console.log('Started node cluster master');
+  logger.info('Started node cluster master');
 } // startMaster
 
+function initApp() {
+  initKafkaLoggingProducer().then(start).catch((err) => {
+    logger.error(err);
+  });
+}
+
```

Comment:
same comment as other PR - I think the toggle check should be done here so it's clear we're only initializing the kafka producer when the toggle is enabled.